src/Core/Translation/Export/TranslationCatalogueExporter.php

```diff
@@ -194,26 +206,10 @@ private function transformCatalogueMapToMessageCatalogue(Catalogue $catalogue, s
     protected function renameCatalogues(string $locale, string $path): void
     {
         $finder = Finder::create();
-
         foreach ($finder->in($path . DIRECTORY_SEPARATOR . $locale)->files() as $file) {
-            $filenameParts = explode('.', $file->getFilename());
-            unset($filenameParts[count($filenameParts) - 1]); // Remove the extension
-            /*
-             * The destination file name will have the format
-             * DIRECTORY/ab-AB/FullDomainName.ab-AB.xlf
```

Comment:
Isn't this comment quite helpful? You like adding comments to the code :D

Reply:
Yes, I love them, added. :-)